src/core/dbus-mount.c

```diff
@@ -86,22 +86,11 @@ static int property_get_type(
                 sd_bus_message *reply,
                 void *userdata,
                 sd_bus_error *error) {
-
-        Mount *m = userdata;
```

Comment:
Please keep that. This tells the reader that `userdata` **must** be of type `Mount`. The compiler is happy with a cast from `void *` later on, but for human readers it's better to make this explicit.